.gitignore

```diff
@@ -1,10 +1,30 @@
-# Do not track objects or archives
```

Comment:
This line shouldn't need to be changed. I can add the commit reverting this change, if that's okay.